server/utils.go

```diff
@@ -23,10 +23,16 @@ import (
 	"github.com/flashbots/mev-boost/config"
 )
 
+const (
+	// DefaultMaxRetries is the default number of retries for HTTP requests
+	DefaultMaxRetries = 5
```

Comment:
Do you want this to be a configurable option? To me, default implies there's a way to change it.